yaml.go

```diff
@@ -113,6 +114,13 @@ func (dec *Decoder) KnownFields(enable bool) {
 	dec.knownFields = enable
 }
 
+// FallbackToJSON enables using the "json" struct tags as a fallback
+// when no "yaml" struct tag is present.
+// If enabled, the anonymous struct fields are also considered for inlining.
+func (dec *Decoder) FallbackToJSON(enable bool) {
```

Comment:
Except this that I would prefer not to be added here, but via an Option passed as a variadic, I'm fine with the changes